### ISSUE TITLE
[Bug]: Cyclic Item Transfer Node deletes items on transfer.

### ISSUE THREAD

**Issue by archieonic** [AllTheMods/ATM-6#3038]
### Possible Fixes

Yes

### Modpack Version

1.8.22

### What happened?

Upon using an Item Transfer Node from Cyclic, most of the items get deleted upon transfer. The steps to reproduce are the normal usage steps for the item in question:

1. Place down Item Transfer Node
2. Use GPS card to record location of container to receive items (in my case, I used a vanilla chest)
3. Put a stack of any item in the node.
4. Go check the chest, and more than 60% of the items are gone.

I checked in Cyclic's discord, and it does not seem to be a bug that happens with Cyclic itself if used on its own.

### latest.log

https://gist.github.com/archieonic/805b6ddffb65d6b6e0f67e397aa3fd47

**Comment by radiomike** [comment]
Does it still happen with optifine removed?

**Comment by archieonic** [comment]
Yes, I rent a server and had a friend test it (he doesn't have optifine) and same result.

**Comment by oly2o6** [comment]
What are you transfering to, and what are you tranporting from (start and end, chest to refined storage for example), and can you replicate this by changing one of the 2 containers

**Comment by archieonic** [comment]
Vanilla chests to vanilla chests was what I first did and where I noticed the bug. Seems to happen if the receiving inventory has more than 1 slot, so any other storage methods like that give the same result. If, like the node, the receiving inventory has only 1 inventory slot/1 free slot, it works as intended.

**Comment by 0xCarti** [comment]
I'm experiencing a weird issue where if the receiving inventory has more then one spot to store items, like a chest for example, the transfer node won't stack transferred items, instead it will transfer 3 of said item into one of the chest slots then transfer another 3 of the same item into the adjacent chest slot. It does this until the receiving inventory slots are all taken up then stops transferring all together. If I go and condense the inventory of the receiving block to make space for more item the node does not resume transferring unless I remove the GPS and re-insert it.
